src/parser.rs

```diff
@@ -5369,11 +5369,15 @@ impl<'a> Parser<'a> {
         }
         let projection = if let Some((align, by)) = align {
             let fill = fill.unwrap_or(String::new());
+            let by_num = FunctionArg::Unnamed(FunctionArgExpr::Expr(Expr::Value(
+                Value::SingleQuotedString(by.len().to_string()),
+            )));
```

Comment:
Why `SingleQuotedString` instead of `Number`?

Reply:
Because `SingleQuotedString` and `Number`'s underlying storage are both `String`. But I'm not sure what kind of scalar type the datafusion will convert  `Number` to, a `i8` or a `u8` or something else, maybe it's more appropriate to do it on my own